src/Compilers/CSharp/Test/Emit3/ExtensionTypeTests.cs

```diff
@@ -21571,7 +21571,7 @@ public static void M(dynamic d)
         var model = comp.GetSemanticModel(tree);
         var memberAccess = GetSyntax<ElementAccessExpressionSyntax>(tree, "(new C())[d]");
         Assert.Equal("System.Int32 C.this[System.Int32 i] { get; }", model.GetSymbolInfo(memberAccess).Symbol.ToTestDisplayString());
-        Assert.Equal(["System.Int32 C.this[System.Int32 i] { get; }"], model.GetMemberGroup(memberAccess).ToTestDisplayStrings());
+        Assert.Equal([], model.GetMemberGroup(memberAccess).ToTestDisplayStrings());
```

Comment:
Why has this changed?

Reply:
That's the way this test without the extension now behaves in `main` branch where some changes relative to `dynamic` were made.
Given that this scenario doesn't actually involve the extension member, I didn't investigate further.

Reply:
> That's the way this test without the extension now behaves in `main` branch where some changes relative to `dynamic` were made. Given that this scenario doesn't actually involve the extension member, I didn't investigate further.

The behavior looks consistent with early bound case (no `dynamic` involved):
```
        [Fact]
        public void Test()
        {
            var source = """
C.M(1);
class C
{
    public static void M(int d)
    {
        System.Console.Write((new C())[d]);
    }
    public int this[int i] => 42;
}
""";
            var comp = CreateCompilation(source, targetFramework: TargetFramework.Net70);
            CompileAndVerify(comp, expectedOutput: ("42"), verify: Verification.FailsPEVerify).VerifyDiagnostics();
            var tree = comp.SyntaxTrees.First();
            var model = comp.GetSemanticModel(tree);
            var memberAccess = GetSyntax<ElementAccessExpressionSyntax>(tree, "(new C())[d]");
            Assert.Equal("System.Int32 C.this[System.Int32 i] { get; }", model.GetSymbolInfo(memberAccess).Symbol.ToTestDisplayString());
            Assert.Equal([], model.GetMemberGroup(memberAccess).ToTestDisplayStrings());
        }
```